src/client/app/components/UIOptionsComponent.jsx

```diff
@@ -95,16 +95,23 @@ export default class UIOptionsComponent extends React.Component {
 				<div className="radio">
 					<label><input type="radio" name="chartTypes" value={chartTypes.bar} onChange={this.handleChangeChartType} checked={this.props.chartToRender === chartTypes.bar} />Bar</label>
 				</div>
+				<div className="radio">
+					<label><input type="radio" name="chartTypes" value={chartTypes.compare} onChange={this.handleChangeChartType} checked={this.props.chartToRender === chartTypes.compare} />Compare</label>
+				</div>
 				{this.props.chartToRender === chartTypes.bar &&
 				<div>
-					<div className="checkbox">
-						<label><input type="checkbox" onChange={this.handleChangeBarStacking} />Bar stacking</label>
-					</div>
 					<p style={labelStyle}>Bar chart interval (days):</p>
 					<Slider min={1} max={365} value={this.state.barDuration} onChange={this.handleBarDurationChange} onChangeComplete={this.handleBarDurationChangeComplete} />
 				</div>
 				}
+				{this.props.chartToRender === chartTypes.bar &&
+				<div className="checkbox">
+					<label><input type="checkbox" onChange={this.handleChangeBarStacking} />Bar stacking</label>
+				</div>
+				}
+				{this.props.chartToRender !== chartTypes.compare &&
```

Comment:
Sort of pedantic, but you change this to be inclusive to bar and line chart instead of disallowing compare chart.